modules/ipi-install-network-requirements.adoc

```diff
@@ -85,17 +118,22 @@ The following table provides an exemplary embodiment of fully qualified domain n
 [width="100%", cols="3,5,2", options="header"]
 |=====
 | Usage | Host Name | IP
-| API | `api.<cluster_name>.<domain>` | `<ip>`
-| Ingress LB (apps) |  `*.apps.<cluster_name>.<domain>`  | `<ip>`
-| Provisioner node | `provisioner.<cluster_name>.<domain>` | `<ip>`
-| Master-0 | `openshift-master-0.<cluster_name>.<domain>` | `<ip>`
-| Master-1 | `openshift-master-1.<cluster_name>-.<domain>` | `<ip>`
-| Master-2 | `openshift-master-2.<cluster_name>.<domain>` | `<ip>`
-| Worker-0 | `openshift-worker-0.<cluster_name>.<domain>` | `<ip>`
-| Worker-1 | `openshift-worker-1.<cluster_name>.<domain>` | `<ip>`
-| Worker-n | `openshift-worker-n.<cluster_name>.<domain>` | `<ip>`
+| API | `api.<cluster_name>.<base_domain>` | `<ip>`
+| Ingress LB (apps) |  `*.apps.<cluster_name>.<base_domain>`  | `<ip>`
+| Provisioner node | `provisioner.<cluster_name>.<base_domain>` | `<ip>`
+| Master-0 | `openshift-master-0.<cluster_name>.<base_domain>` | `<ip>`
+| Master-1 | `openshift-master-1.<cluster_name>-.<base_domain>` | `<ip>`
+| Master-2 | `openshift-master-2.<cluster_name>.<base_domain>` | `<ip>`
+| Worker-0 | `openshift-worker-0.<cluster_name>.<base_domain>` | `<ip>`
+| Worker-1 | `openshift-worker-1.<cluster_name>.<base_domain>` | `<ip>`
+| Worker-n | `openshift-worker-n.<cluster_name>.<base_domain>` | `<ip>`
 |=====
 
+[NOTE]
+====
+Reverse DNS resolution is required for the Kubernetes API, the provisioner node, the control plane nodes, and the worker nodes when using DNS records to set the hostnames instead of DHCP.
```

Comment:
I'm likely being picky here, but I am thinking about translation.  Should this be _Reverse DNS resolution, instead of DHCP, is required..._. 
The way it is now makes it seem like _when using DNS records to set the hostnames instead of setting DHCP_. 
Or maybe that is correct?

Reply:
Reworded the sentence. Basically, the installer will use DHCP reservations to retrieve the hostnames. If they aren't set, the installer will try to use reverse DNS.